### ISSUE TITLE
optional format for the datetime blueprint fields

### ISSUE THREAD

**Issue by aoloe** [getgrav/grav-plugin-admin#1546]
it should be possible to define what the datetime field contains:

https://learn.getgrav.org/forms/blueprints/fields-available#special-form-fields-available-exclusively-in-the-admin

in my case i wanted it to only contain the date (and since the undocumented date field does not behave the same, i'd really prefer to use the datetime field).

i'm pretty sure that it should be trivial to pass the format to the kendo calendar.(if that calendar is still in use).

**Comment by rhukster** [comment]
Agreed, it would be nice to have a time, or date, or both datetime.

**Comment by georgewilli-am** [comment]
:wave:!

Has this been fixed with https://github.com/getgrav/grav-plugin-admin/commit/d6afa396248e1572b0de33eb78ab030b0cc2ec08? I think this is just missing some documentation now. Was looking for a good issue to start work on, so if this needs additional work or a documentation PR, happy to work on either!

**Comment by w00fz** [comment]
You are correct, the date field should take a `format` attribute in the blueprint definition and it takes PHP date format.

If you’d like to test it still works as expected and create a PR for learn where this feature is explained that would be greatly appreciated. It does look like it’s missing: https://learn.getgrav.org/16/forms/blueprints/fields-available#datetime-field

**Comment by georgewilli-am** [comment]
Tested, seems to work OK, I've updated the documentation and submitted a PR.
